modules/rostests/rosautotest/main.cpp

```diff
@@ -100,6 +102,22 @@ wmain(int argc, wchar_t* argv[])
             WineTest.Run();
         }
 
+        /* Clear the stringstream */
+        ss.str("");
+        ss.clear();
+
+        /* Show the beginning time */
+        ss << "[ROSAUTOTEST] System uptime at start was " << setprecision(2) << fixed;
```

Comment:
Are the 3 time needed? Do they need to be on 3 separate lines? And/Or should 'start' time be written at actual start?

Reply:
Even assuming it does not hurt to report the 3 values, 1 log line should be enough for that purpose.